server/test/user/user.e2e-spec.ts

```diff
@@ -0,0 +1,297 @@
+import {HttpModule, INestApplication} from '@nestjs/common';
+import {Test} from '@nestjs/testing';
+import request from 'supertest';
+import {UserModule} from '../../src/port/module/user.module';
+import {User} from '../../src/domain/user/user.entity';
+import {DatabaseModule} from '../../src/port/module/database.module';
+import {ConfigModule, ConfigService} from 'nestjs-config';
+import * as path from 'path';
+import {getConnection} from 'typeorm';
+import * as assert from 'assert';
+
+// todo: implement ExceptionHandler, apply custom Error
+
+describe('UserController', () => {
+    let app: INestApplication;
+    const initUserForGet = {id: 1, address: 'firstAddress', name: 'firstName', point: 0, level: 0};
+    const initUserForIncreaseLevelAndPoint = {id: 2, address: 'secondAddress', name: 'secondName', point: 0, level: 0};
+    const initUserForDecreasePoint = {id: 3, address: 'thirdAddress', name: 'thirdName', point: 10, level: 0};
+    const initUserForDelete = {id: 4, address: 'fourthAddress', name: 'fourthName', point: 0, level: 0};
+
+    beforeAll(async () => {
```

Comment:
in e2e test cases we don't need to restart whole app per tests!!

Reply:
didn't understand. it's 'beforeAll', not 'beforeEach'